.gitignore

```diff
@@ -141,3 +141,6 @@ dmypy.json
 .envrc
 .env.ecr
 *.sqlc
+
+# Typescript
+tsconfig.tsbuildinfo
```

Comment:
a quick research shows that `tsbuildinfo` is not worth checking in 

https://github.com/vercel/next.js/discussions/30815